googletest/src/gtest.cc

```diff
@@ -2979,13 +2979,15 @@ void ColoredPrintf(GTestColor color, const char* fmt, ...) {
   CONSOLE_SCREEN_BUFFER_INFO buffer_info;
   GetConsoleScreenBufferInfo(stdout_handle, &buffer_info);
   const WORD old_color_attrs = buffer_info.wAttributes;
-
+  // Let's reuse the BG
+  const WORD existing_bg = old_color_attrs & 0x00F0;
```

Comment:
I'm uncomfortable with the `0xf0` magic mask. Can this be programmatically determined?
Does MSVC provide a mask constant to apply here?